### ISSUE TITLE
In SVG diagrams, MatrixGate does not render

### ISSUE THREAD

**Issue by Strilanc** [quantumlib/Cirq#5690]
```
import cirq
from cirq.contrib.svg import SVGCircuit

with open('tmp.svg', 'w') as f:
    print(SVGCircuit(cirq.Circuit(
        cirq.MatrixGate(cirq.unitary(cirq.X)).on(cirq.LineQubit(0)),
    ))._repr_svg_(), file=f)
```

![image](https://user-images.githubusercontent.com/79941/178052137-0e132948-0f25-4f12-b89c-76c50c8e23a0.png)

**Comment by dabacon** [comment]
Duplicate of #4630

**Comment by vtomole** [comment]
@dabacon You linked an unmerged PR, not an issue. It looks like this issue will be closed once that PR is merged.

**Comment by dabacon** [comment]
Ooops thanks @vtomole

**Comment by dabacon** [comment]
Proper duplicate is #2313
